Makefile.e2e.ci

```diff
@@ -8,11 +8,18 @@
 test-e2e-gke-ci: pull-postsubmit-retry test-e2e-gke-nobuild
 
 POSTSUBMIT_GCS_PREFIX ?= gs://kpt-config-sync-ci-postsubmit
-POSTSUBMIT_REGISTRY ?= us-docker.pkg.dev/kpt-config-sync-ci-artifacts/postsubmit
+POSTSUBMIT_REGISTRY ?= $(LOCATION)-docker.pkg.dev/$(TEST_INFRA_PROJECT)/postsubmit
 
 .PHONY: postsubmit
-postsubmit: build-cli
```

Comment:
I think `postsubmit` still depends on `build-cli` as `publish-gcs` publishes both manifests and nomos CLI.

Reply:
Yeah this was an accident that got left in from local testing, see https://github.com/GoogleContainerTools/kpt-config-sync/pull/1163